packages/adapter-cloudflare-workers/index.js

```diff
@@ -51,13 +54,18 @@ export default function () {
 
 			await esbuild.build({
 				entryPoints: [`${tmp}/entry.js`],
-				outfile: `${entrypoint}/index.js`,
-				bundle: true,
+				outfile: `${entrypoint}/${main_path}`,
 				target: 'es2020',
-				platform: 'browser'
+				platform: 'browser',
+				bundle: true,
+				external: ['__STATIC_CONTENT_MANIFEST'],
```

Comment:
One thing I'd like to see from #4576 is the ability to pass in `esbuild` options, which is something `adapter-cloudflare` supports. We occasionally need to mark external dependencies for legacy libraries that reference Node utils, or provide polyfills.

Reply:
added and rebased.  cf adapter allows overriding `platform` and `target` so it is allowed here too or should it be fixed for workers?